docs/upgrade/3.10.1.rst

```diff
@@ -0,0 +1,82 @@
+.. _upgrade-to-3.10.1:
+
+####################
+3.10.1 release notes
+####################
+
+This release focuses on Python 3.10, node 16 (for build system), and bug fixes.
+
+********************
+What's new in 3.10.1
+********************
+
+Features:
+=========
+* Add support for Django 3.2 LTS version
+* Page changed_date added to the Page tree admin actions dropdown template #6701 (#7046) (73cbbdb00) -- Vladimir Kuvandjiev
+* Allow recursive template extending in placeholders (#6564) (fed6fe54d) -- Stefan Wehrmeyer
+* Added ability to set placeholder global limit on children only (#6847) (18e146495) -- G3RB3N
+* Replaced Travis.CI with Github Actions (#7000) (0f33b5839) -- Vinit Kumar
+* Added support for Github Actions based CI.
+* Added Support for testing frontend, docs, test and linting in different/parallel CI pipelines.
+* Added django-treebeard 4.5.1 support, previously pinned django-treebeard<4.5 to avoid breaking changes introduced
+* Improved performance of ``cms list plugins`` command
+* Page changed date added to the Page tree admin actions dropdown
+
+Bug Fixes:
+==========
+
+* using .nvmrc to target the right nvm version (3e5227def) -- Florian Delizy
+* Fixed an issue where the wrong page title was returned (#6466) (3a0c4d26e) -- Alexandre Joly
+* Add toolbar fix for broken CMS in the release 3.10.x -- Vinit Kumar
+* Fixed #6413: migrations 0019 and 0020 on multi db setups (#6708) (826d57f0f) -- Petr Glotov
+* Added fix to migrations to handle multi database routing (#6721) (98658a909) -- Michael Anckaert
+* Fixed issue where default fallbacks is not used when it's an empty list (#6795) (5d21fa5eb) -- Arjan de Pooter
+* Fixed prefix_default_language = False redirect behavior (#6851) (34a26bd1b) -- Radek Stępień
+* Fix not checking slug uniqueness on page move (#6958) (5976d393a) -- Iacopo Spalletti
+* Fixed DontUsePageAttributeWarning message (#6734) (45383888e) -- carmenkow
+* Fixed Cache not invalidated when using a PlaceholderField outside the CMS #6912 (#6956) (3ce63d7d3) -- Benjamin PIERRE
+* Fixed unexpected behavior get_page_from_request (#6974) (#6073) (52f926e0d) -- Yuriy Mamaev
+* Fixed django treebeard 4.5.1 compatibility (#6988) (eeb86fd70) -- Aiky30
+* Fixed Bad Title.path in Multilanguage sites if parent slug is created or modified (#6968) (6e7b0ae48) -- fp4code
+* Fixed redirect issues when i18n_patterns had prefix_default_language = False
+* Fixed not checking slug uniqueness when moving a page
+* Fixed builds on RTD
+* Fixed the cache not being invalidated when updating a PlaceholderField in a custom model
+* Fixed 66622 bad Title.path in multilingual sites when parent slug is created or modified
+* Fixed 6973 bag with unexpected behavior ``get_page_from_request``
```

Comment:
Question: would this refers to issue #6973?

Reply:
Quite likely